TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project.component.ts

```diff
@@ -18,19 +19,23 @@ import { PercentPipe } from '@angular/common';
 })
 export class ProjectComponent implements OnInit {
   projects: Project[];
-
+  isAdmin:boolean;
```

Comment:
Would be great if components would use service's method or var instead of separate local variables

Reply:
I wrote class methods that call service methods instead of local vars.  I suppose that is what you meant